### ISSUE TITLE
Non matching regex !~ doesn't work with SHOW TAG VALUES

### ISSUE THREAD

**Issue by sofixa** [influxdata/influxdb#7338]
### Bug report

__System info:__ InfluxDB 0.13.0 under Debian Jessie(8.3)

__Steps to reproduce:__

1. Using matching regex with SHOW TAG VALUES 
2. Using not matching(negative?) regex with SHOW TAG VALUES
3. Sadness

__Expected behavior:__ For the regex to work, as per the docs

__Actual behavior:__ It works just fine when it's matching, but not the other way around

__Additional info:__ Sample data:

```
> SHOW TAG VALUES from nfsv3 with KEY="node"
name: nfsv3
-----------
key     value
node    pa1-01a
node    pa1-01b
node    pa3-01a
node    pa3-01b
node    pa2-01b
node    pa2-01a
node    pa2-02b
node    pa2-02a
node    pa2-03a
node    pa2-03b

```

Working matching regex:

```
> SHOW TAG VALUES from nfsv3 with KEY="node" WHERE node =~ /pa2/
name: nfsv3
-----------
key     value
node    pa2-01b
node    pa2-01a
node    pa2-02b
node    pa2-02a
node    pa2-03a
node    pa2-03b


> SHOW TAG VALUES from nfsv3 with KEY="node" WHERE node =~ /.*b.*/
name: nfsv3
-----------
key     value
node    pa1-01b
node    pa3-01b
node    pa2-01b
node    pa2-02b
node    pa2-03b


```

Non working non matching regex(tried with a single letter as well, as per the [docs here](https://docs.influxdata.com/influxdb/v0.13/query_language/data_exploration/#regular-expressions-and-specifying-tags) ):

```
> SHOW TAG VALUES from nfsv3 with KEY="node" WHERE node !~ /pa2/
> SHOW TAG VALUES from nfsv3 with KEY="node" WHERE node !~ /.*b.*/
>

```

Was non matching regex support removed at some point or am i going blind?

Thanks in advance,

Cheers,
Adrian

**Comment by jsternberg** [comment]
Can you try this with 1.0.0? I think I remember fixing a bug related to this for 0.13. I'll try to find the issue.

**Comment by jsternberg** [comment]
Looks like it was found in #6607 and fixed with #6616.
